src/arvbuffer.c

```diff
@@ -372,6 +372,11 @@ arv_buffer_get_frame_id (ArvBuffer *buffer)
 	return buffer->priv->frame_id;
 }
 
+void arv_buffer_set_frame_id(ArvBuffer *buffer, guint32 frameId)
+{
+  buffer->priv->frame_id=frameId;
```

Comment:
Hi. The indentation in other code seems to be tabs of width 8. It would be nicer to read the file if it were consistent.